include/LCObjects/LCTrack.h

```diff
@@ -0,0 +1,215 @@
+/**
+ * @file LCContent/LCObjects/include/LCTrack.h
+ *
+ * @brief Header file for LC Track class
+ *
+ */
+
+#ifndef LC_TRACK_H
+#define LC_TRACK_H 1
+
+#include "Objects/Track.h"
+#include "Api/PandoraApi.h"
+
+#include "Pandora/ObjectCreation.h"
+#include "Pandora/PandoraObjectFactories.h"
+
+#include "Persistency/BinaryFileReader.h"
+#include "Persistency/BinaryFileWriter.h"
+#include "Persistency/XmlFileReader.h"
+#include "Persistency/XmlFileWriter.h"
+
+
+namespace lc_content
+{
+
+typedef std::vector<pandora::InputTrackState> LCInputTrackStates;
+typedef std::vector<pandora::TrackState> LCTrackStates;
+
+
+/**
+ *  @brief  LCTrack Parameters, allow multiple track states at the calorimeter
+ */
+class LCTrackParameters : public object_creation::Track::Parameters
+{
+public:
+    LCInputTrackStates m_trackStates;       ///< Vector of TrackStates
+};
+
+
+
+class LCTrack: public object_creation::Track::Object
```

Comment:
Doxygen, whitespace (not going to push too hard on this, though)!

Reply:
Added `@brief` doxygen comment, removed some empty lines